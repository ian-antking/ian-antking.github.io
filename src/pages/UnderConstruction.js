import React from 'react';
import styled from 'styled-components';
import { ReactComponent as UnderConstructionImage } from '../assets/undraw_under_construction_46pa.svg';

const StyledImage = styled(UnderConstructionImage)`
  width: 75vw;
  height: 75vw;
`;

const UnderConstruction = () => (
  <>
    <StyledImage />
    <h2>Under Construction</h2>
  </>
);

export default UnderConstruction;
