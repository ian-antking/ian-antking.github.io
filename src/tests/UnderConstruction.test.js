import React from 'react';
import { render } from '@testing-library/react';
import UnderConstruction from '../pages/UnderConstruction';

describe('UnderConstruction', () => {
  it('renders the correct snapshot', () => {
    const { asFragment } = render(<UnderConstruction />);

    expect(asFragment()).toMatchSnapshot();
  });
});
