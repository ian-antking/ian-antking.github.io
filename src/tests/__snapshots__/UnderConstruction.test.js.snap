// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UnderConstruction renders the correct snapshot 1`] = `
<DocumentFragment>
  <svg
    class="sc-AxjAm eSIdJy"
  >
    undraw_under_construction_46pa.svg
  </svg>
  <h2>
    Under Construction
  </h2>
</DocumentFragment>
`;
